service/controller/v22/encrypter/vault/vault.go

```diff
@@ -152,21 +152,16 @@ func (e *Encrypter) EnsureCreatedAuthorizedIAMRoles(ctx context.Context, customO
 		return microerror.Mask(err)
 	}
 
-	ctlCtx, err := controllercontext.FromContext(ctx)
+	cc, err := controllercontext.FromContext(ctx)
```

Comment:
We just named it `ctlCtx` in @tomahawk28 PR. Would be good to get some consensus in fmt.

Reply:
https://github.com/giantswarm/fmt/pull/42

Reply:
42 <3  I voted for `ctlCtx`.

Reply:
We can change once we know how.